### ISSUE TITLE
Multi-Account Containers interfering with 1Password non-deterministically 

### ISSUE THREAD

**Issue by CamJN** [mozilla/multi-account-containers#1423]
- [x] "Firefox will: Never remember history" in the Firefox Preferences/Options under "Privacy & Security > History" is NOT selected
- [x] You are NOT using Firefox in a Private Window
- [x] You can see a greyed out but ticked Checkbox with the description "Enable Container Tabs" in the Firefox Preferences/Options under "Tabs"

- Multi-Account Containers Version: 6.1.0
- Operating System + Version: macOS 10.14.5 (18F203)
- Firefox Version: 68.0b3 (Dev edition)
- Other installed Add-ons + Version + Enabled/Disabled-Status:

|Name|Version|Enabled|ID|
|---|---|---|---|
|1Password extension (desktop app required)|4.7.3.90|true|onepassword4@agilebits.com|
|About Sync|0.0.50|true|aboutsync@mhammond.github.com|
|Amazon.co.uk|1.1|true|amazon@search.mozilla.org|
|Bing|1.0|true|bing@search.mozilla.org|
|DuckDuckGo|1.0|true|ddg@search.mozilla.org|
|eBay|1.0|true|ebay@search.mozilla.org|
|Google|1.0|true|google@search.mozilla.org|
|hotfix-update-xpi-intermediate|1.0.6|true|hotfix-update-xpi-intermediate@mozilla.com|
|React Developer Tools|3.6.0|true|@react-devtools|
|Simple Tab Groups|3.4.4|true|simple-tab-groups@drive4ik|
|Twitter|1.0|true|twitter@search.mozilla.org|
|uBlock Origin|1.19.2|true|uBlock0@raymondhill.net|
|Wikipedia (en)|1.0|true|wikipedia@search.mozilla.org|
|Firefox Multi-Account Containers|6.1.0|true|@testpilot-containers|

### Actual behaviour
Frequently, but not every time, 1Password is unable to communicate with its browser extension and thinks that it isn't installed. Disabling and then re-enabling this extension will fix it for a while when it happens.

### Expected behaviour
For 1Password to be able to communicate with the browser extension.

### Steps to reproduce
1. Install & setup 1Password
2. Install 1Password Firefox browser extension
3. Install Multi-Account Containers extension
4. Create container (or switch to a pre-created named container)
5. Press command+\ or click the 1Password button
6. Note 1Password is unaware that the 1Password Firefox browser extension is installed
7. Disable Multi-Account Containers extension
8. Press command+\ or click the 1Password button
9. Note 1Password functions normally

### Notes
1Password has refused to do anything about this, so hopefully it can be fixed from this side.

**Comment by CamJN** [comment]
New info, it only works to have both MAC enabled alongside 1password if 1password is loaded by firefox first, is there a way to control this? Right now i'm either having to get lucky or disable MAC when starting firefox then enable it afterwards once 1password has already loaded.

**Comment by ksmithut** [comment]
@CamJN Is this similar to this issue? https://github.com/mozilla/multi-account-containers/issues/1454

**Comment by CamJN** [comment]
@ksmithut no, that issue seems to be about a keyboard shortcut conflicting, whereas this bug is about 1password somehow not being able to tell what the url of the current site is, when the 1password extension is loaded after the MAC extension by firefox.

I expect it's due to the MAC extension somehow isolating the 1password extension from the tabs. Maybe the 1password extension gets stuck in a container? I don't know but it's not a shortcut issue, as I can call up 1password fine, it just doesn't know what page I'm on.
